packages/ubuntu_desktop_installer/lib/pages/allocate_disk_space/allocate_disk_space_dialogs.dart

```diff
@@ -8,6 +8,35 @@ import 'package:ubuntu_wizard/widgets.dart';
 import '../../l10n.dart';
 import 'allocate_disk_space_model.dart';
 
+/// Shows a confirmation dialog with the given title and message.
+Future<bool> showConfirmationDialog(
+  BuildContext context, {
+  required String title,
+  required String message,
+}) async {
+  final lang = UbuntuLocalizations.of(context);
+  final result = await showDialog<bool>(
+    context: context,
+    builder: (context) {
+      return AlertDialog(
+        title: Text(title),
+        content: Text(message),
+        actions: <Widget>[
+          OutlinedButton(
+            child: Text(lang.backAction),
+            onPressed: () => Navigator.of(context).pop(false),
+          ),
+          OutlinedButton(
+            child: Text(lang.continueAction),
+            onPressed: () => Navigator.of(context).pop(true),
+          ),
+        ],
+      );
+    },
+  );
+  return result == true;
```

Comment:
`result` being a boolean, this is redundant. You could just return `result`.

Reply:
It can be null when dismissing the dialog by clicking outside of it.